raft/src/test/java/org/apache/kafka/raft/RaftEventSimulationTest.java

```diff
@@ -1014,6 +1015,26 @@ public void verify() {
         }
     }
 
+    private static class LeaderNeverLoadSnapshot implements Invariant {
+        final Cluster cluster;
+
+        private LeaderNeverLoadSnapshot(Cluster cluster) {
+            this.cluster = cluster;
+        }
+
+        @Override
+        public void verify() {
+            for (RaftNode raftNode : cluster.running()) {
+                if (raftNode.counter.isWritable()) {
+                    assertTrue(raftNode.counter.getHandleSnapshotCalls() == 0);
```

Comment:
Let's use `assertEquals`:
```suggestion
                    assertEquals(0, raftNode.counter.getHandleSnapshotCalls());
```

Reply:
done